chanzuckerberg/single-cell-explorer
# feat: move continuous fields to the dataset drawer if there is a single value for all cells

#615 

Adds redux state and handling logic in histogram component to remove fields if the anno matrix is not a view and the field contains a single value for all obs.

To test:

copy the test dataset to `example-dataset/`: 
```sh
AWS_PROFILE=single-cell-prod aws s3 cp --recursive s3://hosted-cellxgene-prod/67a79be6-e85f-471b-82cf-1d773f61f8f2.cxg ./singleton.cxg/
```

navigate to the dataset

you should not see `donor_BMI_at_collection` in the LSB, but rather the dataset drawer
